src/guide/quickstart.md

```diff
@@ -8,6 +8,8 @@ gz_size: "29.40"
 ro_gz_size: "20.70"
 ---
 
+<p class="tip">MachineLabs is currently in private beta. <strong>To follow this quickstart you need to be part of the private beta program</strong>. If you haven't done so yet, login at <a href="https://machinelabs.ai/editor" title="MachineLabs Editor">machinelabs.ai</a> to become part of the next batch. We onboard new users weekly.</p>
```

Comment:
Here we speak directly *to* the user. (If you haven't done so yet). I think I like that more than speaking about them (see my previous comment about the intro changes)

Reply:
Yes this is on purpose because it's a straight message to the reader who's currently reading this document. It's before the journey in the quickstart starts (where we're sitting with the reader in the same boat, which is why we use "we" and "us" everywhere).

Reply:
In fact this hint will be removed again once we're in public beta, so it's not really part of the documentation, it's really just a temporary hint for the user. We don't use "you" anywhere. At least, i think we shouldn't and just stick to one (which is "we", "us" so far)

Reply:
I'm ok with `we` and `us`. I don't like it much when we speak *about* the users though (see my comments in the introduction) because the users are the audience that read the document so it feels a bit distanced to speak about them. I'd rather use `we` and/or `us` there, too.